myocamlbuild.ml

```diff
@@ -6,10 +6,10 @@ open Ocamlbuild_plugin;;
 
 let env = BaseEnvLight.load() (* setup.data *)
 
-let fortran =
-  try BaseEnvLight.var_get "fortran" env with _ -> failwith "XXX"
+let fortran = try BaseEnvLight.var_get "fortran" env with _ -> failwith "XXX"
```

Comment:
A holdout from when I was looking through the file, I found the one liner more readable. If you don't like the change, happy to remove it.